cloud-haven-react-app/src/index.js

```diff
@@ -4,6 +4,10 @@ import { BrowserRouter } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/js/bootstrap.js';
```

Comment:
I'm a little confused about this part. I thought we were using react-bootstrap, which is a bit easier/cleaner/easier to use than regular bootstrap imo

Reply:
that's my mistake, i'll fix it

Reply:
i've changed it to react-bootstrap